examples/geos-cbm/SamVlirHdr.c

```diff
@@ -0,0 +1,43 @@
+/********************************************************************
+ * SamVlir.h
+ *
+ * This file contains the header block definition for the llvm-mos
+ * GEOS VLIR sample application.
+ *
+ ********************************************************************/
+
+#include <geos.h>
+
+#include "SamVlir.h"
+
+__attribute__((section(".info_block"))) 
+__attribute__((retain)) 
+file_header_t __std_file_header = {
+    3,
```

Comment:
These struct data items could use comments with the field name for any where it isn't immediately clear from the value provided (e.g. contrast APPLICATION and 3).

Reply:
Its a fixed value (number of bytes for one pixel row of an icon). It is documented at the type (file_header_t) and in the GEOS programming documentation, so i feel, i will not repeat it here...